app/views/repos/new.html.slim

```diff
@@ -1,6 +1,3 @@
-- @repo.errors.full_messages.each do |msg|
-  li= msg
-
 - if user_signed_in?
```

Comment:
I removed that and transferred the showing of errors in the controller through flash.now[:error] so that it will use the bootstrap's design for flashing error messages.